samples/java/src/main/java/br/com/example/leadIntegration/controllers/LeadIntegrationController.java

```diff
@@ -0,0 +1,19 @@
+package br.com.example.leadIntegration.controllers;
+
+import org.springframework.web.bind.annotation.*;
+import br.com.example.leadIntegration.models.Lead;
+
+@RestController
+@RequestMapping("/leads")
+public class LeadIntegrationController {
+
+    @GetMapping("/health-check")
+    public String healthCheck() {
+        return "{\"message\": \"OK\"}";
+    }
+
+    @PostMapping("/lead")
+    public String lead(@RequestBody Lead lead){
```

Comment:
Do you are checking the `SECRET_KEY? :thinking: 

something like this: `https://github.com/olxbr/crm-lead-integration/blob/master/samples/python/app/api.py#L19`